zboxcore/sdk/chunked_upload.go

```diff
@@ -594,12 +597,27 @@ func (su *ChunkedUpload) processUpload(chunkStartIndex, chunkEndIndex int,
 			defer wg.Done()
 			err = b.sendUploadRequest(ctx, su, chunkEndIndex, isFinal, encryptedKey, body, formData, pos)
 			if err != nil {
+				select {
+					case wgErrors <- err:
+					default:
+				}
+				wgErrors <- err
```

Comment:
@Manish-210 please remove this `wgErrors <- err`, it will cause the goroutine stuck.

Reply:
Corrected.